pkg/pdf/pdf_test.go

```diff
@@ -1531,6 +1531,13 @@ func newMarotoTest(fpdf *mocks.Pdf, math *mocks.Math, font *mocks.Font, text *mo
 	return m
 }
 
+func basePdfHelperTest(left, top, right float64) *mocks.Pdf {
```

Comment:
Is this how it should be tested?

Reply:
These private functions are to arrange the dependencies to allow us to unit test some code. We already have one (basePdfTest) to create the *mocks.Pdf, why don´t you use it? Just add you SetMargins to it :)

Reply:
Good point will do! How would you go about mocking GetMargins? Now that they can be set. Should the method take the margins like my new function did, and just move the SetMargins mock in?